packages/nextcloud/test/notes_test.dart

```diff
@@ -153,13 +153,18 @@ void main() {
       });
 
       test('Get and update settings', () async {
+        final expectedSettings = notes.Settings(
+          (b) => b
+            ..notesPath = 'Notes'
+            ..fileSuffix = '.md'
+            ..noteMode = notes.Settings_NoteMode.rich,
+        );
+
         var response = await client.notes.getSettings();
         expect(response.statusCode, 200);
         expect(() => response.headers, isA<void>());
 
-        expect(response.body.notesPath, 'Notes');
-        expect(response.body.fileSuffix, '.md');
-        expect(response.body.noteMode, notes.Settings_NoteMode.rich);
+        expect(response.body, expectedSettings);
```

Comment:
```suggestion
        expect(response.body, equalsBuilt(expectedSettings));
```

This produces better error messages in case the test fails.